### PR TITLE
Updating cluster computer slide presenter notes.

### PR DESCRIPTION
**Pull request by lcoghill**
Updating presenter notes for cluster computer that were duplicated from the "cloud computing" slides. Attempting to address issue [419](https://github.com/carpentries-incubator/hpc-intro/issues/419).

### CODE REVIEW

File: files/jargon.html [hunk 1]
```diff
@@ -146,14 +146,10 @@
...
 
-Cloud systems physically resemble clusters or supercomputers, but
-with greater heterogeneity among components, and less coupling, often
-without an interconnect at all. Cloud servers are generally accessed
-by connecting to their services over the network. Multiple virtual
-machines may reside on individual physical machines. Typical tasks
-are driven by user connections from the outside to the various "front
-end" services (e.g. web sites) which pass traffic on to the "back
-end" servers (e.g. databases) to present results to users.
+A collection of standalone computers that are networked together. The will
```
Comment (reid-a) [inline files/jargon.html:149]: At the end of line 149 here, I think "The will" should be "They will"?

**Review by reid-a — CHANGES_REQUESTED** [review]
Grammar error on line 149 -- hopefully an easy fix!

**Comment by reid-a** [comment]
I have made a separate PR, #423, which addresses the CI build error -- once that's merged into the main repo, and incorporated into this PR, things should work.

**Comment by reid-a** [comment]
I have taken the liberty of updating the branch upstream to incorporate the spelling correction -- once the requested changes have been made, this PR should be good to merge, assuming the CI runs successfully.

**Comment by reid-a** [comment]
I should say, once the requested changes have been *addressed* -- you don't have to do what was asked, but you do need to either make the case for the existing version, or propose other changes. Thanks again for contributing!

File: files/jargon.html (hunk 1 above)
Comment (bkmgit) [reply files/jargon.html:149]: Ok

**Comment by reid-a** [comment]
I have taken the liberty of making my requested grammar fix directly on the branch. Will merge ASAP.

Should not be taken to detract from the value of this contribution, this is terrific!

**Review by reid-a — APPROVED** [review]

**Pull request merged**